pages/views-11ty/layouts/basic.njk

```diff
@@ -11,7 +11,7 @@
       } else if (!('getRootNode' in Element.prototype) || !window.customElements || !window.IntersectionObserver) {
         version = 'light';
       }
-      (version) && document.write('<script type="text/javascript" src="/bundles/polyfill-' + version + '.js"><\/script>');
+      (version) && document.write(`<script type="text/javascript" src="{{ '/bundles/polyfill-${version}.js' | url }}"><\/script>`);
```

Comment:
Template strings are part of ES6 syntax, so there is no reason to load polyfill for IE with the way that IE doesn't recognize ))

Reply:
it's funny! thanks, I didn't think about it